src/styles/components/_related-components.scss

```diff
@@ -34,9 +32,9 @@ $breakpoint-move-arrow-left: 38.125rem; // 610px/16
       background-image: url("../../../public/images/cmt_inventory_arrow.svg");
       position: absolute;
       left: 2rem;
-      top: -4.5rem;
-      width: 3.75rem; // 60px/16
-      height: 4.6875rem; // 75px/16
+      top: -5rem;
+      width: 3.5rem; // 60px/16
+      height: 4.2rem; // 75px/16
```

Comment:
I was running into this too! Would you update the comments with the correct new values? 3.5rem is 56px, and 4.2rem is ~67px.